packages/core/src/components/detail/detail.scss

```diff
@@ -0,0 +1,74 @@
+:host {
```

Comment:
Set `box-sizing: border-box;`. The ShadowDOM prevents the global style from leaking through